prog-o-meter.py

```diff
@@ -51,48 +52,54 @@ def __init__(self, _days, _filename, _username):
         self.days = _days
         self.GOAL = 100
         self.rectangle_list = []
+        self.encourage = ["Nice Job!", "The force is strong with you!", "Well Done!", "You are doing it!", "Never, never, never give up.", "If you dream it, you can do it.", "Everything you can imagine is real.", "Hope is a waking dream.", "You are the best!", "Always believe in yourself."]
         self.days_remaining = self.GOAL - self.days
         self.completion_date = self.get_completion_date(self.days_remaining-1)
-        # Tkinter instantiation 
+        self.x = 0

```

Comment:
Since there are many people working on this code, we need to make it easy to understand what is happening throughout the code. To do this, it is important that it is clear what every variable does. Is there a name that better describes this variable that we can use?